external_tool_compensation: [1.131, -1.01, -2.5, -0.171, -0.142, 0.252]


# RSS: replay:
# cartesian_stiffness_target: [300, 300, 300, 50, 50, 50]
# cartesian_stiffness_target: [300, 300, 500, 50, 70, 50]
cartesian_stiffness_target: [0, 0, 0, 0, 0, 0]

# RSS: teach:
# nullspace_stiffness_target: [10, 5, 0.1, 0.1, 0.1, 0.000001, 0.1]
# RSS: execute
nullspace_stiffness_target: [0.1, 0.1, 0.01, 0.01, 0.01, 0.01, 0.01]
# nullspace_stiffness_target: [0.00001, 1, 50, 0.05, 5, 0.05, 1]
